### DISCUSSION TITLE
Snap camera device doesn't appear on Windows 11

### DISCUSSION THREAD

**Discussion by elzibubble** [ptrumpis/snap-camera-server#136] [Q&A]
Hi, fresh Win11 PC, trying to get Snap Camera working again. Installed reqs + project, all up in docker, client finds my camera and applies lens, all great. However no "snap camera" device is made available through Windows, I can't see it in "Settings" app camera list, or the "Camera" app device dropdown or in Discord settings. Halp please.

**Comment by elzibubble** [comment]
I've also tried running Snap Camera as admin, doesn't help.

**Comment by Jbla0712** [comment]
Hi, do you use Splitcam software? 
Try to uninstal it or other virtual camera software before installing snapcamera.
You need disable media foundation frame server in your windows. He often gave me problems

![eg](https://github.com/ptrumpis/snap-camera-server/assets/13618418/a599daed-152b-4379-b429-ef896a21a732)
This is your problem with it (i think)
This prevents the use of multiple virtual cameras. I'm runing Win 11 too

**Comment by elzibubble** [comment]
I don't use any other camera software, fresh Win11 PC. I tried adding that key and restarting, didn't help. Additionally I checked Task Manager and I couldn't see anything running which was reminiscent of "Windows Media Foundation" or "Service Host: Frame Server".
